### DISCUSSION TITLE
Blank Space in the bottom

### DISCUSSION THREAD

**Discussion by tumon-art** [LazyVim/LazyVim#4933] [General]
I'm using latest version. 
It was working fine before. I tried different terminals but same issue.
![image](https://github.com/user-attachments/assets/2574e252-b4f8-4c24-9cd1-5e4b163fa8db)

**Comment by folke** [comment]
What is the exact issue here?
Either way, please update snacks and check if it's fixed.

**Reply by tumon-art** [reply]
This is the the issue, too big blank space in the bottom.
![image](https://github.com/user-attachments/assets/40fdc6fe-638e-46bf-a872-e8cabf1dc62b)

**Reply by folke** [reply]
Can't reproduce this with stock LazyVim.
You probably have a custom statusline or something like that that loads on the dashboard?
Snacks disables statusluine and tabline, but when another plugin then overwrites it, there's little I can do...

**Reply by tumon-art** [reply]
**I tried reinstalling it without adding new things of my own.** 
I still see the same issue for some specific window sizes. I think space is getting bigger on the right side as well now after updating to latest.

![image](https://github.com/user-attachments/assets/76783b6e-5cdb-4bba-b7a0-a2b760766985)

**Reply by folke** [reply]
That's just your terminal size and you don't have terminal padding.
Your terminal width/height is not an exact multiple of cell width/height
